pkg/apis/resource/extension.go

```diff
@@ -304,10 +304,20 @@ func (h Handler) getEndpointsFromOutput(ctx context.Context, id object.ID) ([]Ac
 				Name:      v.Name,
 			})
 		case v.Schema.Type == openapi3.TypeArray:
-			if v.Schema.Items == nil || v.Schema.Items.Value == nil ||
-				v.Schema.Items.Value.Type != openapi3.TypeString {
-				// For list and set: all elements are the same type.
-				return nil, invalidTypeErr
+			if v.Schema.Items != nil && v.Schema.Items.Value != nil {
+				// Array.
+				if v.Schema.Items.Value.Type != "" && v.Schema.Items.Value.Type != openapi3.TypeString {
```

Comment:
When `v.Schema.Items.Value.Type != openapi3.TypeString`, it must not equal to `""`, So it just need to change to 
```
if v.Schema.Items.Value.Type != openapi3.TypeString {
    xxx
}
```

Reply:
when the type isn't set, also allows to pass